src/main/java/build/buildfarm/instance/shard/JedisClusterFactory.java

```diff
@@ -208,6 +209,10 @@ private static Supplier<UnifiedJedis> createJedisClusterFactory(
   private static ConnectionPoolConfig createConnectionPoolConfig() {
     ConnectionPoolConfig connectionPoolConfig = new ConnectionPoolConfig();
     connectionPoolConfig.setMaxTotal(configs.getBackplane().getJedisPoolMaxTotal());
+    connectionPoolConfig.setMaxIdle(configs.getBackplane().getJedisPoolMaxIdle());
+    connectionPoolConfig.setMinIdle(configs.getBackplane().getJedisPoolMinIdle());
+    connectionPoolConfig.setTimeBetweenEvictionRuns(
```

Comment:
You mentioned to set this to `-1` to disable the behavior. Does this need a `-1` check here, or does Jedis/Apache Commons Pool recognize a -1 Millis Duration?

Reply:
Since Apache Commons Pool2 sets -1 as the default value, I believe there is no need for Buildfarm to perform positive/negative checks.

See https://github.com/apache/commons-pool/blob/rel/commons-pool-2.12.0/src/main/java/org/apache/commons/pool2/impl/BaseObjectPoolConfig.java

```java
public static final long DEFAULT_TIME_BETWEEN_EVICTION_RUNS_MILLIS = -1L;

public static final Duration DEFAULT_DURATION_BETWEEN_EVICTION_RUNS = Duration.ofMillis(DEFAULT_TIME_BETWEEN_EVICTION_RUNS_MILLIS);

public void setTimeBetweenEvictionRuns(final Duration timeBetweenEvictionRuns) {
    this.durationBetweenEvictionRuns = PoolImplUtils.nonNull(timeBetweenEvictionRuns, DEFAULT_DURATION_BETWEEN_EVICTION_RUNS);
}
```